pom.xml

```diff
@@ -148,6 +148,15 @@
           <groupId>${tycho-groupid}</groupId>
           <artifactId>tycho-compiler-plugin</artifactId>
           <version>${tycho-version}</version>
+          <configuration>
+            <extraClasspathElements>
+              <extraClasspathElement>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
+                <version>1.2.8</version>
```

Comment:
Please use a property for that version, so we don't need to sync bundles/pom.xml and pom.xml.